slam-libraries/viam-orb-slam3/orbslam_server_v1_test.cc

```diff
@@ -247,13 +277,35 @@ BOOST_AUTO_TEST_CASE(
 
 BOOST_AUTO_TEST_CASE(
     ParseAndValidateArguments_offline_with_no_delete_processed_data) {
+    const vector<string> args{
+        "-data_dir=/path/to",  "-config_param={mode=rgbd}", "-port=20000",
+        "-sensors=",           "-data_rate_ms=200",         "-map_rate_sec=60",
+        "-use_live_data=false"};
+    SLAMServiceImpl slamService;
+    utils::ParseAndValidateArguments(args, slamService);
+    BOOST_TEST(slamService.offlineFlag == true);
```

Comment:
[minor] We should replace `slamService.offline_flag` with `slamService.use_live_data` and test all four cases (`use_live_data` vs. `sensors`).

Reply:
all tested!